### ISSUE TITLE
Zoom "end" is not called when pinch zooming and you start the zoom with the 2nd finger outside the chart

### ISSUE THREAD

**Issue by VaggV** [d3/d3-zoom#259]
This also happens sometimes when the 2 fingers are inside the container at the start of the zoom (and the 2nd is outside when zooming stops), but it happens consistently if I start zooming with the 2nd finger outside the container.

I was using version 7.3.0 but when I tested with 7.8.0 it had the same behavior.

**Comment by Fil** [comment]
is this a duplicate of #229?

**Comment by VaggV** [comment]
No, I have tried this on a tablet on Chrome and Samsung Internet (and the behavior is the one I am describing). I will edit my original post
